src/components/uielements/label/label.style.ts

```diff
@@ -1,8 +1,17 @@
 import styled from 'styled-components'
 import { palette, key } from 'styled-theme'
 
-import { Props } from './label'
-import { FontSettings, Colors } from './types'
+import { Size, Color, FontSettings, Colors } from './types'
+
+type Props = {
+  size?: Size
+  color?: Color
+  weight?: string
+  className?: string
+  children?: React.ReactNode
+  loading?: boolean
+  onClick?: () => void
+}
```

Comment:
Define `Props` for `label.style` only, which are really used for styles. For example:

```
type Props = {
  size?: Size
  color?: Color
  weight?: string
  usePointer?: boolean
}
```

Reply:
Fixed!